src/katgpucbf/fgpu/compute.py

```diff
@@ -116,7 +116,7 @@ def __init__(
         spectra: int,
         spectra_per_heap: int,
     ) -> None:
-        self.dig_sample_bits = template.pfb_fir.dig_sample_bits
+        self.dig_sample_bits = template.pfb_fir.input_sample_bits
```

Comment:
> If you're going to rename the parameter of the pfb_fir, it makes some sense to rename the above in the "compute", earlier in the file.

I renamed the pfb_fir parameter because it does not always refer to the number of bits in digitiser samples; in `complex_input` mode it is instead the number of bits that come out of the DDC. On the other hand, I wanted this attribute to refer to the bits in digitiser samples regardless.

> If the data is needed, it can just be accessed as self.template.pfb_fir.input_sample_bits anyway.

Once the DDC is wired in, it becomes more complicated. For wideband it will be `self.template.pfb_fir.input_sample_bits` but for narrowband it will be `self.template.ddc.dig_sample_bits`.

> It doesn't look as though this member variable actually ever gets used. Perhaps it can be removed?

That's a reasonable point - I'm not sure why I have it. Will try deleting it and see what happens.

Reply:
It looks like it can be removed, but doing so on this PR will cause merge conflicts with later work I've done. If you don't object, I'll rather remove it in a future PR.